src/start_server.ts

```diff
@@ -113,6 +116,22 @@ export function getApp(options: ServerOptions) {
   return app;
 }
 
+/**
+ * Open the given web page URL. If no browser keyword is provided, `opn` will use
+ * the user's default browser.
+ */
+function openWebPage(url: string, withBrowser?: string) {
+  let openOptions = {
+    app: withBrowser
+  };
+  opn(url, openOptions, (err) => {
+    if (err) {
+      console.log(`Error launching "${openOptions.app || 'default web browser'}".`);
```

Comment:
I wonder if we want to return the error, rather than throw, so that it can be collected in the loop below and thrown after trying to launch all browsers. That way we don't block subsequent browsers from opening if one fails. This, of course, is totally minor relative to fixing the behavior here, and could be done later.